### ISSUE TITLE
FM clarity

### ISSUE THREAD

**Issue by mochet** [Joystream/marketing#80]
Suggestion (raised in the RU telegram):
- for each FM up until incentives v3 launch, add a one line description of why they were inducted

Reason:
- Transparency and setting in stone the current system once it is over.

**Comment by bedeho** [comment]
@blrhc Perhaps we can extend the schema to record some of this information on `Persons`, at least for new FMs, however it would be significant work to do this retrospectively. I am not sure I am fully understanding what the problem is, are we saying people in the future will hesitate to join because of not knowing the history?

I guess I could add a `History` section to https://joystream.gitbook.io/testnet-workspace/key-concepts/founding-member-program.

**Comment by mochet** [comment]
> @blrhc Perhaps we can extend the schema to record some of this information on `Persons`, at least for new FMs, however it would be significant work to do this retrospectively. I am not sure I am fully understanding what the problem is, are we saying people in the future will hesitate to join because of not knowing the history?
> 
> I guess I could add a `History` section to https://joystream.gitbook.io/testnet-workspace/key-concepts/founding-member-program.

Yes, the issue is two things
- During the testnet people should be able to see and understand why people were given FM status, so they can have a goal or idea of why people were inducted
- After mainnet launch and at any point in the future the reason should be stated so that there is no question of why people were inducted. A lot of history for answering this currently is on our testnet forums, community repo and possibly other areas, so it is important to ensure that even without access to this data that people can understand why.

We must especially consider the mindset of outsiders joining the project who have no understanding of anyone participating so far, they may see relationships rather than contributions and it is important for them to be able to understand the "why" of the FM program. Ideally it should maybe include links to things the users have done (which is difficult considering subsequent chains are going to lose on-chain content)

**Comment by bedeho** [comment]
> During the testnet people should be able to see and understand why people were given FM status, so they can have a goal or idea of why people were inducted

Yes, so this we can solve by both adding it the the profile of each user, and also to make it more clear when actually announcing FMs. Made an issue about this here https://github.com/Joystream/marketing/issues/89.

> After mainnet launch and at any point in the future the reason should be stated so that there is no question of why people were inducted. A lot of history for answering this currently is on our testnet forums, community repo and possibly other areas, so it is important to ensure that even without access to this data that people can understand why.

This one is harder, mainly because @blrhc has to go and recall his old judgements over a very long period of time, but lets see what we can do, made an issue here: https://github.com/Joystream/marketing/issues/90

**Comment by mochet** [comment]
The community in the RU telegram are doing an initiative for current FMs to explain what they did to get FM status. This is a great initiative from them, but JSG should also still give its rationale.

**Comment by bedeho** [comment]
That is awesome! If they write it up, Ben can verify and use it as input.

**Comment by mochet** [comment]
> That is awesome! If they write it up, Ben can verify and use it as input.

Ok great, I will let them know.

@bedeho 
I think my overarching point here for "why" to do this and to emphasize why it is important:
* I think of the `genesis distribution` less like a `genesis block` and much more like `immaculate conception`.
* When we launch mainnet, all of this information should be recorded and anyone with questions should be able to see the reasons for why FMs were inducted.
* Without explanation, its just a mysery, and if things are not recorded, documented and explained ahead of time, things get lost or there can be other questions. To keep it short: if it isn't handled this way, it is messy, disorganized and **expensive**.
* After mainnet, if anyone raises any question, there should be a link pointing to a list of who was inducted at this stage in the project and why--questions are answered + it was already publicly shared months ago.
* The FM program "now" should be signed, sealed and delivered with reasons for user's induction in a reasonable timeframe. It should not happen when mainnet launches, it should be done now, the link shared in Discord/Twitter or a blog post which is something akin to _"The first phase of the FM program has now finished, this is the summary of what happened"_. Clearly explaning things, providing a link to all of the FM summaries (https://github.com/Joystream/founding-members/blob/main/scoring-periods/1.md). It should be "concluded".

**Comment by CheOmsk** [comment]
@bedeho Andy has created a spreadsheet and we are filling it in: https://docs.google.com/spreadsheets/d/139Xzv7VRgz2yV5m5aFViLK00M1wYOaoqXy9hfLN2Dso/edit?usp=sharing
